### DISCUSSION TITLE
What is exactly the "globe" projection?

### DISCUSSION THREAD

**Discussion by jjimenezshaw** [maplibre/maplibre-gl-js#5019] [Q&A]
In this post https://floss.social/@bart/113424884295371526 Bart mentions the globe-to-mercator projection :+1: 

I would like to know how it is working.

- Is the "globe" part like this?
ref: https://github.com/d3/d3-geo-projection?tab=readme-ov-file#geoSatellite
implementation code: https://github.com/d3/d3-geo-projection/blob/main/src/satellite.js
example: https://observablehq.com/@d3/satellite-explorer (maplibre does not use the tilt I guess)
- if not, what is the "projection" behind?
- is there any transition to the Mercator projection (like a linear interpolation or whatever)

Thanks

PS. I tried to understand the code in https://github.com/maplibre/maplibre-gl-js/blob/main/src/geo/projection/globe_transform.ts but I am not that fluent in that environment to find the math behind.

**Comment by HarelM** [comment]
This is currently in "discussion" here:
https://github.com/maplibre/maplibre-style-spec/pull/888

We are making the projection specification a bit more clear, hopefully.
